crossctf-quals2018/CYOA/README.md

```diff
@@ -0,0 +1,65 @@
+# Choose Your Own Adventure - CrossCTF Quals 2018 (misc)
+
+> Maybe your question was too long. You *reduced* the length of your question and asked the oracle again: "Where's the flag?"
+> As expected, she had no *reaction* again. You sigh... maybe the *solution* really was in there after all. You close your eyes and started willing yourself back into the enemy's base...
+> Non-standard flag format: <code>[A-Z0-9]+</code>
+
+## Challenge
+We are given the hints in this format:
+
+>01 01 02, Score: 04
```

Comment:
Add newlines for the few lines below

Reply:
Do you want writeups for RSA 1 and 2 too?

Reply:
Sure thanks